render/mapping.go

```diff
@@ -276,7 +307,10 @@ func MapContainer2ContainerImage(n RenderableNode) (RenderableNode, bool) {
 		return n, false
 	}
 
-	return newDerivedNode(id, n), true
+	// Add container-<id> key to NMD, which will later be counted to produce the minor label
+	result := newDerivedNode(id, n)
+	result.NodeMetadata.Metadata[containersPrefix+n.ID] = ""
+	return result, true
```

Comment:
For the record: we decided offline to pursue the separate counters struct in the NodeMetadata.